app/models/subscription.rb

```diff
@@ -58,7 +58,9 @@ def subscribe!(brand, subscription_plan, email, stripe_token)
       if customer
         stripe_subscription_id = customer.subscriptions.first.id
         create_payment_handler!(brand, customer)
-        create_subscription!(brand, subscription_plan, stripe_subscription_id, trial_end)
+        subscription = create_subscription!(brand, subscription_plan, stripe_subscription_id, trial_end)
+        TransactionalEmail.welcome(brand).send
```

Comment:
Not needed for now but it would be nice to send these off to a sidekiq worker. We might have some mailers that may need to make a bunch of db calls that would slow down this transaction. It would be nice if we could have this done in a worker.